src/drt/src/serialization.h

```diff
@@ -56,6 +56,9 @@
 #include "global.h"
 #include "odb/dbTypes.h"
 #include "odb/geom.h"
+
+using uint = odb::uint;
```

Comment:
You putting this back into the global namespace.

Reply:
Good catch- will handle